### DISCUSSION TITLE
Issue with airodump-ng pwr values

### DISCUSSION THREAD

**Discussion by juliefavre** [aircrack-ng/aircrack-ng#2569] [Q&A]
Hello everyone!

I am using aircrack-ng on a Oneplus 7, with Kali Nethunter 2023.2 installed on the top of OxygenOs 10 (Android 10).

So far it works, but I have an issue : when I use airodump-ng, the signal power is not properly shown, it returns pwr 94db for all the wifi spots. When I use it through another software such as wifite I have the same issue, the pwr is 6db this time for all the wifi spots.

Otherwise aircrack-ng seems to work properly, and the wifi card seems well to be in onitor mode. The firmwares installed are in the package kali-linux-firmware, and the firmware in use seems to be "icnss".

Did someone already got this issue of bad values of power? Have you any idea of what could cause this?

Kind regards

**Comment by Mister-X-** [comment]
I don't use NetHunter, but these values come from the driver. It's likely the driver developers didn't care about filling the correct signal, so you just have bogus values.

What driver are you using?
What version of aircrack-ng are you running? Don't say latest, give the value.

**Reply by juliefavre** [reply]
The driver is "icnss", and the aircrack-ng version is 1.7 rev 24075023
